digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetOperations.scala

```diff
@@ -310,12 +310,24 @@ trait PointAssetOperations {
     }
   }
 
+  def getPersistedAssetsByIdsWithExpire(ids: Set[Long]): Seq[PersistedAsset] = {
+    withDynSession {
+      getPersistedAssetsByIdsWithExpireWithoutTransaction(ids)
+    }
+  }
+
```

Comment:
some of that methods was created for trafficSignManager, please check if it continues to be used

Reply:
Yes, we continue using this method.